build.gradle

```diff
@@ -35,6 +35,9 @@ buildscript {
 
 allprojects {
 	repositories {
+		flatDir {
```

Comment:
This is the only thing I haven't seen before. Pretty straight forward, the docs: https://docs.gradle.org/current/userguide/declaring_repositories.html state that this is how to specify a location in the source code to pull/store the dependencies (linking in case anyone else is curious)